packages/dd-trace/test/opentracing/propagation/text_map.spec.js

```diff
@@ -434,8 +436,18 @@ describe('TextMapPropagator', () => {
         delete textMap['x-datadog-parent-id']
       })
 
+      it('should skip extraction the header', () => {
```

Comment:
The names of these tests are not clear. I had to read them to understand what they are actually testing. Let's make sure the title includes the context, for example "should skip extraction when disabled".

Reply:
this is based on b3 ones: https://github.com/DataDog/dd-trace-js/blob/5081f444208fdf4685c7cebe30da92950f698da0/packages/dd-trace/test/opentracing/propagation/text_map.spec.js#L283-L382

Reply:
The B3 test name is `should skip extraction without the feature flag`. Something equivalent would definitely work to make it clear that this is testing without the feature flag.